core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -376,7 +399,7 @@ var StoryEditorPage = function() {
     // by the navbar.
     await general.scrollToTop();
     var chapterTitleButton = await chapterTitles.get(index);
-    await chapterTitleButton.click();
+    await action.click('Chapter Title Button', chapterTitleButton);
   };
 
   this.expectNodeOutlineToMatch = function(nodeOutline) {
```

Comment:
Can you add waitFor statements before we interact with elements in `this.expectNodeOutlineToMatch` and `this.expectExplorationIdAlreadyExistWarning` below?